packages/Python/lldbsuite/test/lang/swift/member_lookup_crash/main.swift

```diff
@@ -10,7 +10,8 @@ extension Measurement where UnitType == UnitAngle {
   }
 
   func f() {
-    return //%self.expect('p self.radians', substrs=["property 'radians' requires the types 'UnitType' and 'UnitAngle' be equivalent"], error=True)
+    return //%self.expect('p self.radians', substrs=["(CGFloat) $R0 = 1.745"])
+           //%self.expect('p self', substrs=["Measurement<UnitAngle>"])
```

Comment:
Do you think it would be possible to extract the interesting bits from the type and remove the dependency on Foundation? It would make it much clearer what is being tested for.

Reply:
Maybe, but this is testcase reported by somebody internally so I would love to keep it as is.
[we can add another one]

Reply:
If you want to keep it, you could duplicate the test into
1. an end-to-end test inside the foundation subdirectory
2. a coverage test that faithfully reproduces the bug fixed by this commit without any dependencies